taxi_service/settings.py

```diff
@@ -42,6 +42,7 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "crispy_bootstrap4",
     "debug_toolbar",
```

Comment:
The `debug_toolbar` is included in the `INSTALLED_APPS` and `MIDDLEWARE`. This is useful for development but should be removed or disabled in a production environment to avoid performance overhead and potential security risks.